namespace-provisioner/use-case7.hbs.md

```diff
@@ -0,0 +1,471 @@
+# Work with Git repositories in air-gapped with Namespace Provisioner
```

Comment:
suggestion: in air-gapped -> in air-gapped environments?

Reply:
Changed